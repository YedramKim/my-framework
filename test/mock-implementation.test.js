jest.mock('./module');
const mod = require('./module');

test('mockImplementation 테스트', function() {
	mod.mockImplementation(_ => 100);
	expect(mod()).toBe(100);
});

it('스냅샷 테스트 가즈아아아아아', () => {
	expect('<div>스냅샷 테스트 GAZUAAAAA~~</div>').toMatchSnapshot();
});