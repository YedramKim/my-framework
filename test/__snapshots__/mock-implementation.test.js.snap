// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`스냅샷 테스트 가즈아아아아아 1`] = `"<div>스냅샷 테스트 GAZUAAAAA~~</div>"`;
